offline/QA/Jet/ConstituentsinJets.h

```diff
@@ -64,6 +85,9 @@ class ConstituentsinJets : public SubsysReco
   std::string m_recoJetName;
   std::string m_towBkgdName;
   std::string m_histTag;
+  std::string m_towCEMCName = {"TOWERINFO_CALIB_CEMC_RETOWER"};//input node string for EMCAL
+  std::string m_towIHCALName = {"TOWERINFO_CALIB_IHCAL"};//input node string for IHCAL
+  std::string m_towOHCALName = {"TOWERINFO_CALIB_OHCAL"};//input node string for OHCAL
```

Comment:
I'd suggest moving these initializations to the ctor. That way we keep it consistent with the other `std::string` members and will appease clang-tidy.
```suggestion
  std::string m_towCEMCName; //input node string for EMCAL
  std::string m_towIHCALName ; //input node string for IHCAL
  std::string m_towOHCALName; //input node string for OHCAL
```